### PR TITLE
Main > Translations: Add some translations.

### PR DESCRIPTION
**Pull request by gonzalo-plaza**

### CODE REVIEW

**Comment by gonzalo-plaza** [comment]
@Andresws12 @PVillasana I have added some translations in the app.

**Comment by gonzalo-plaza** [comment]
@Andresws12 @PVillasana this is okay to be reviewed.

File: src/components/Utils/NoAppoints.vue [hunk 1]
```diff
@@ -4,11 +4,10 @@
             <section class="hero is-danger">
                 <div class="hero-body">
                     <div class="container">
-                        <h1 class="title">Ninguna cita Asignada</h1>
+                        <h1 class="title">{{ $('utils.noAppointsTitle') }}</h1>
                         <h2 class="subtitle">
                             <router-link to="/inicio"
-                                >Puedes asignarte una cita pulsando
-                                aquí</router-link
+                                >{{ $('utils.noAppointsSubtitle') }}</router-link
```
Comment (Andresws12) [inline src/components/Utils/NoAppoints.vue:10]: @gonzalo-plaza remember add all tree from the translations, components.utils....

File: src/components/Utils/NoAppoints.vue [hunk 2]
```diff
@@ -4,11 +4,10 @@
             <section class="hero is-danger">
                 <div class="hero-body">
                     <div class="container">
-                        <h1 class="title">Ninguna cita Asignada</h1>
+                        <h1 class="title">{{ $('utils.noAppointsTitle') }}</h1>
```
Comment (Andresws12) [inline src/components/Utils/NoAppoints.vue:7]: @gonzalo-plaza remember add all tree from the translations, components.utils....

**Comment by gonzalo-plaza** [comment]
@Andresws12 I have corriged the fails !!

**Pull request merged**